internal/initializer/installer_test.go

```diff
@@ -39,7 +39,7 @@ const (
 	errPatchProviderSourceFmt      = "unexpected source in provider update: %s"
 	errGetConfigurationFmt         = "unexpected name in configuration get: %s"
 	errPatchConfigurationFmt       = "unexpected name in configuration update: %s"
```

Comment:
There is a lot of errors above with `err...Fmt` should they also be changed to `errFmt...`?

Reply:
Good call, regexes 😅